packages/connect/src/api/blockchainEstimateFee.ts

```diff
@@ -61,10 +61,10 @@ export default class BlockchainEstimateFee extends AbstractMethod<'blockchainEst
     async run() {
         const { coinInfo, request } = this.params;
         const feeInfo: MethodReturnType<typeof this.name> = {
-            blockTime: coinInfo.blocktime,
+            blockTime: coinInfo.blockTime,
             minFee: coinInfo.minFee,
             maxFee: coinInfo.maxFee,
-            dustLimit: coinInfo.type === 'bitcoin' ? coinInfo.dustLimit : undefined,
+            dustLimit: coinInfo.type === 'bitcoin' ? coinInfo.dustLimit : -1,
```

Comment:
runtime change right? I understand that non-utxo coins don't have dust limit (what about cardano?). why it did not make sense to use undefined here?

Reply:
you are right, it will be safer to return undefined [18ed620](https://github.com/trezor/trezor-suite/pull/9171/commits/18ed6204d689f69d4f4738545fbc411561043aff)